specutils/spectra/spectral_coordinate.py

```diff
@@ -416,8 +416,8 @@ def radial_velocity(self):
         coordinate frame in that this calculates the radial velocity with
         respect to the *observer*, not the origin of the frame.
         """
-        return np.sum(self._calculate_radial_velocity(
-            self._observer, self._target), axis=0)
+        return self._calculate_radial_velocity(self._observer, self._target,
+                                               as_scalar=True)
```

Comment:
The length of the vector can't be found using ``sum``, and ``linalg.norm`` would return the absolute magnitude, so rather than duplicate code, I now use ``_calculate_radial_velocity`` with the option to return a signed scalar